README.md

```diff
@@ -2,6 +2,8 @@
 Python client for Kraken API REST and Kraken Websockets API using httpx and websockets.
 Supports both sync and async for API REST.
 
+Documentation: https://kraky.readthedocs.io/en/latest/
```

Comment:
In Fastapi repo, they put Documentation in bold : https://github.com/tiangolo/fastapi

If we want people to find the documentation, makeing it bold can certainly help.

Reply:
Can you also rebase on main, I did some changes to the workflow so that it passes ?